src/lib/handlers/keeper.ts

```diff
@@ -1,27 +1,21 @@
-import { MessageEvent } from "../../types/clients/keeper"
-import { Handler } from "../../types/generic";
-import KeeperClient from "../clients/keeper"
+import { Command } from "../../types/generic";
 
-type CommandHandler = Handler<MessageEvent, KeeperClient>;
-
-export const sayPong: CommandHandler = async ({ logger, command: { say, message } }) => {
-    logger.log(`got ping: ${JSON.stringify(message.text)}`);
-    await say(`Pong! -- ${message.text}`);
-}
-
-export const sayHello: CommandHandler = async ({ command: { say, message } }) => {
-    await say(`hello, <@${message.user}>!`);
-}
-
-export default [
+const commands: Command[] = [
     {
         requireMention: true,
         matches: /ping/,
-        handler: sayPong,
+        handler: async ({ logger, command: { say, message } }) => {
```

Comment:
Why this style? Seems a bit more difficult to maintain this way instead of breaking out functions

Reply:
I figured keeping it all local to the command itself was more clear! Less jumping around. I'm open to other opinions here though 😅

Reply:
It's probably not a big deal for now,  especially since most of our handlers will be relatively small.